mmv1/products/alloydb/Instance.yaml

```diff
@@ -13,6 +13,11 @@
 
 --- !ruby/object:Api::Resource
 name: 'Instance'
+docs: !ruby/object:Provider::Terraform::Docs
+  warning: |
+    Deleting an instance with instanceType = SECONDARY does not delete the secondary instance but gives a warning asking to delete the associated secondary cluster as well.
```

Comment:
```suggestion
    Deleting an instance with instanceType = SECONDARY does not delete the secondary instance, and abandons it instead.
```

Reply:
Note that we log a debug log at the WARNING level, which is not the same thing as prompting a user with an interactive warning. Users without debug logging enabled won't see anything.